Apps/Admin/Client/Pages/SupportPage.razor

```diff
@@ -94,95 +105,65 @@
                       Striped="true"
                       Dense="true"
                       data-testid="user-table">
+                <ColGroup>
+                    <MudHidden Breakpoint="@Breakpoint.SmAndDown">
+                        <col />
+                        <col style="min-width: 125px;" />
+                        <col style="min-width: 125px;" />
+                        <col style="min-width: 125px;" />
+                    </MudHidden>
+                </ColGroup>
                 <HeaderContent>
                     <MudTh>
-                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.Hdid)">
-                            HDID
+                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.SortName)" InitialDirection="@SortDirection.Ascending">
+                            Name
                         </MudTableSortLabel>
                     </MudTh>
                     <MudTh>
-                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.PersonalHealthNumber)">
-                            PHN
+                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.Birthdate ?? DateOnly.MinValue)">
+                            DOB
                         </MudTableSortLabel>
                     </MudTh>
                     <MudTh>
-                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.LastLoginDateTime ?? DateTime.MinValue)"
-                                           InitialDirection="@SortDirection.Descending">
-                            Last Login
+                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.PersonalHealthNumber)">
+                            PHN
                         </MudTableSortLabel>
                     </MudTh>
                     <MudTh>
-                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.CreatedDateTime ?? DateTime.MinValue)"
-                                           InitialDirection="@SortDirection.Descending">
-                            Account Created
+                        <MudTableSortLabel SortBy="new Func<PatientRow, object>(x => x.Hdid)">
+                            HDID
                         </MudTableSortLabel>
                     </MudTh>
-                    <MudTh Style="text-align:right"></MudTh>
                 </HeaderContent>
                 <RowTemplate>
-                    <MudTd DataLabel="HDID" data-testid=@($"user-table-hdid-{context.Hdid}")>@context.Hdid</MudTd>
+                    <MudTd DataLabel="Name" data-testid=@($"user-table-name-{context.Hdid}")>
+                        @if (context.Status != PatientStatus.Default)
+                        {
+                            <MudIcon Icon="fas fa-triangle-exclamation" Size="Size.Small" Class="mr-2" data-testid="warning-icon" />
+                        }
+                        @context.Name
```

Comment:
Nit: I would consider putting the icon in its own column (no column name). Keeps the names lined up with and without the icon.

Reply:
If @mk08sh agrees, can include this change with any other tweaks in another task.